WebService/v1/Models/SimulationApiModel/DeviceModelSimulationOverride.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApiModel
+{
+    // SEE: <DeviceModelApiModel.DeviceModelSimulation> for the original fields being overridden
```

Comment:
same comment as devicemodelapimodel subclass above.

Reply:
replied in the other comment